
Waiting for JIT...
Traceback (most recent call last):
  File "/home/pedro/PINN/jaxpi/examples/geomIII2/main.py", line 39, in <module>
    app.run(main)
  File "/usr/local/lib/python3.11/dist-packages/absl/app.py", line 308, in run
    _run_main(main, args)
  File "/usr/local/lib/python3.11/dist-packages/absl/app.py", line 254, in _run_main
    sys.exit(main(argv))
             ^^^^^^^^^^
  File "/home/pedro/PINN/jaxpi/examples/geomIII2/main.py", line 31, in main
    train.train_and_evaluate(FLAGS.config, FLAGS.workdir)
  File "/home/pedro/PINN/jaxpi/examples/geomIII2/train.py", line 114, in train_and_evaluate
    model.state = model.update_weights(model.state, batch)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/jax/_src/traceback_util.py", line 179, in reraise_with_filtered_traceback
    return fun(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/jax/_src/api.py", line 1787, in cache_miss
    out = map_bind_continuation(execute(*tracers))
                                ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/jax/_src/profiler.py", line 335, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/dist-packages/jax/_src/interpreters/pxla.py", line 1213, in __call__
    results = self.xla_executable.execute_sharded(input_bufs)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt